include/SW_Flow_lib.h

```diff
@@ -170,30 +170,21 @@ void SW_ST_init_run(ST_RGR_VALUES* StRegValues);
 
 void SW_ST_setup_run(

```

Comment:
Using `*SW_Site` as argument for `SW_ST_setup_run()` is much nicer than all the individual parameters! Thanks to getting rid of the soil layer info struct!